docs/runtimes/configuring-job-resources.md

```diff
@@ -263,16 +256,8 @@ fn.with_node_selection(node_selector={"app.iguazio.com/lifecycle":"non-preemptib
 
```

Comment:
These two lines should come before the `train_fn.run()` line.
Also, they use `fn` rather than `train_fn` like the rest of the code - should be modified.